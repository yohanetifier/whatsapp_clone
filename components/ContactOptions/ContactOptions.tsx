import React from 'react';

interface Props {}

const ContactOptions = (props: Props) => {
	return <div data-testid="contactOptions">ContactOptions</div>;
};

export default ContactOptions;
