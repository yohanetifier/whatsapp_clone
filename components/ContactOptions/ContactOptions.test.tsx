import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import ContactOptions from './ContactOptions';

describe('ContactOptions', () => {
	it('should render 4 icons  ', () => {
		const { asFragment } = render(<ContactOptions />);
		expect(asFragment()).toMatchSnapshot();
	});
});
