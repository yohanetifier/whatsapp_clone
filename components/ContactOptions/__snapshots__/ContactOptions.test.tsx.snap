// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactOptions should render 4 icons   1`] = `
<DocumentFragment>
  <div
    class="flex justify-between w-[200px]"
    data-testid="contactOptions"
  >
    <svg
      class="w-[28px] h-[28px]"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0z"
        fill="none"
      />
      <path
        d="M20 15.5c-1.25 0-2.45-.2-3.57-.57a1.02 1.02 0 0 0-1.02.24l-2.2 2.2a15.045 15.045 0 0 1-6.59-6.59l2.2-2.21a.96.96 0 0 0 .25-1A11.36 11.36 0 0 1 8.5 4c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.5c0-.55-.45-1-1-1zM19 12h2a9 9 0 0 0-9-9v2c3.87 0 7 3.13 7 7zm-4 0h2c0-2.76-2.24-5-5-5v2c1.66 0 3 1.34 3 3z"
      />
    </svg>
    <svg
      class="w-[28px] h-[28px]"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 16 16"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 5a2 2 0 0 1 2-2h7.5a2 2 0 0 1 1.983 1.738l3.11-1.382A1 1 0 0 1 16 4.269v7.462a1 1 0 0 1-1.406.913l-3.111-1.382A2 2 0 0 1 9.5 13H2a2 2 0 0 1-2-2zm11.5 5.175 3.5 1.556V4.269l-3.5 1.556zM2 4a1 1 0 0 0-1 1v6a1 1 0 0 0 1 1h7.5a1 1 0 0 0 1-1V5a1 1 0 0 0-1-1z"
        fill-rule="evenodd"
      />
    </svg>
    <svg
      class="w-[28px] h-[28px]"
      fill="currentColor"
      height="1em"
      id="search"
      stroke="currentColor"
      stroke-width="0"
      version="1.1"
      viewBox="0 0 24 24"
      width="1em"
      x="0px"
      xmlns="http://www.w3.org/2000/svg"
      y="0px"
    >
      <g>
        <path
          d="M20.031,20.79c0.46,0.46,1.17-0.25,0.71-0.7l-3.75-3.76c1.27-1.41,2.04-3.27,2.04-5.31
		c0-4.39-3.57-7.96-7.96-7.96s-7.96,3.57-7.96,7.96c0,4.39,3.57,7.96,7.96,7.96c1.98,0,3.81-0.73,5.21-1.94L20.031,20.79z
		 M4.11,11.02c0-3.84,3.13-6.96,6.96-6.96c3.84,0,6.96,3.12,6.96,6.96c0,3.84-3.12,6.96-6.96,6.96C7.24,17.98,4.11,14.86,4.11,11.02
		z"
        />
      </g>
    </svg>
    <svg
      class="w-[28px] h-[28px] ml-4"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      stroke-width="0"
      viewBox="0 0 512 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M256 294.1L383 167c9.4-9.4 24.6-9.4 33.9 0s9.3 24.6 0 34L273 345c-9.1 9.1-23.7 9.3-33.1.7L95 201.1c-4.7-4.7-7-10.9-7-17s2.3-12.3 7-17c9.4-9.4 24.6-9.4 33.9 0l127.1 127z"
      />
    </svg>
  </div>
</DocumentFragment>
`;
